data/json/items/gun/9mm.json

```diff
@@ -740,11 +740,11 @@
       [ "grip", 1 ],
```

Comment:
Shouldn't this also be turned into a mount 🤔

Reply:
Well the Luty does already have a grip, though perhaps not provisions to make it easily replaceable I suppose, so eh, sure, why not?